src/components/molecules/forms/loginForm/LoginForm.tsx

```diff
@@ -11,7 +11,16 @@ interface LocationState {
   from: { pathname: string };
 }
 
-const LoginForm = () => {
+/* shouldRegister determines whether 'bli medlem' will render */
+export interface LoginFormProps {
+  shouldRedirect: boolean;
```

Comment:
an idea could be to have a prop called `redriect_url?: string | undefined` and if a url is provided the form redirects